cms/djangoapps/contentstore/api/views/course_validation.py

```diff
@@ -207,7 +208,8 @@ def _certificates_validation(self, course):
         is_activated, certificates = CertificateManager.is_activated(course)
         return dict(
             is_activated=is_activated,
-            has_certificate=len(certificates) > 0,
+            has_certificate=certificates is not None and len(certificates) > 0,
+            is_enabled=has_certificates_enabled(course),
```

Comment:
I had been under the impression that `is_enabled` would be leveraging the `certificates is not None` logic as well, but could have remembered incorrectly. To make sure that I understand the logic correctly, can you explain the difference in the check done in this method vs the `is not None` route? Thank you!

Reply:
@fysheets I didn't get your question of the differences you asked between. Can you explain? thanks.

Reply:
Sure thing! I'm wondering why you're using `is_enabled=has_certificates_enabled(course)`, rather than checking if `is_enabled=certificates is not None`? I was under the impression that if `certificates is None` that means it is not enabled on that domain, and if they are enabled generally then `certificates = []`. I didn't think we would need the `has_certificates_enabled` function, and I wanted to check my understanding of the logic.

Reply:
Copy that. I think now I understand what you mean. I think it makes sense to use `is_enabled=certificates is not None` rather than calling the new method. 

Let me make this change.

Reply:
Awesome! Thank you